tests/expected_tests.cc

```diff
@@ -0,0 +1,39 @@
+#include "gtest/gtest.h"
+#include "stout/expected.h"
+
+stout::expected<int, std::string> divide(int divident, int divider) {
+  if (!divider) {
+    return stout::make_unexpected("divide by zero");
+  } else {
```

Comment:
You can drop the `else`.

So here:

```c++
if (divider == 0) {
  return stout::make_unexpected("divide by zero");
}
return divident / divider;
```

Inspiration: https://testing.googleblog.com/2017/06/code-health-reduce-nesting-reduce.html